Supported Wi-Fi bands [HC-1542] (#383)
arduino/help-center-content: content/Arduino Cloud/General Cloud/Supported-Wi-Fi-bands.md

---

> title: "Supported Wi-Fi bands"

Reviewer: ```suggestion
title: "Supported Wi-Fi bands for Arduino devices"
```
Adding a bit more context to the title.

id: 14144656051740
---

> ## Use 2.4 GHz Band for IoT Devices

Reviewer: ```suggestion
```
I would remove this header and use the sentence in line 8 as an introduction to the article.

Arduino boards and most IoT devices can only be used with 2.4 GHz band networks. The 2.4 GHz band offers better range and signal strength through obstacles compared to 5 GHz.

> ## How to Check if Your Network is 2.4 GHz

Reviewer: ```suggestion
## How to check if your network is 2.4 GHz
```
Lowercase to match other headings

> Follow these steps to identify the frequency band of your Wi-Fi.

Reviewer: ```suggestion
Follow these steps to identify the frequency band of your Wi-Fi network.
```

**Windows:**

1. Open the settings menu in Windows and select the **Network and Internet** section.

1. Under the wireless network settings, click on the **Hardware properties** link.

1. A properties page will open. Here, you can find details about the wireless network, including the network band (GHz).

**macOS:**

> 1. Click on the Apple menu in the top-left corner of your screen and select **System Preferences**.
> 
> 1. In the System Preferences window, click on the **Network** icon.
> 
> 1. Select the Wi-Fi connection from the list on the left-hand side. Click on the **Advanced** button in the bottom-right corner.
> 
> 1. In the **Wi-Fi** tab, you'll find information about the Wi-Fi connection, including the frequency band (GHz).

Reviewer: ```suggestion
1. Connect to the Wi-Fi network.

1. Click on the Wi-Fi icon on the right side of the top navigation bar while holding down the Option/Alt key.

1. Look for the entry starting with Channel. This will specify if the network is in the 2.4 GHz or 5 GHz band.

```
Updated instructions for macOS

> ## Creating a 2.4 GHz Network with a Hotspot

Reviewer: ```suggestion
## Creating a 2.4 GHz network with a hotspot
```
Lowercase to match other headings
